Media_Varianza_Desconocida.py

```diff
@@ -9,4 +9,58 @@
     </summary>
 """
 def menu_varianza_desconocida():
-    return
\ No newline at end of file
+    print("Elige la prueba de hipotesis a utilizar:")
+    print("1.")
+    print("H0 " + Smu + " = " + Smu + "0")
+    print("H1 " + Smu + " < " + Smu + "0")
+    print("2.")
+    print("H0 " + Smu + " = " + Smu + "0")
+    print("H1 " + Smu + " > " + Smu + "0")
+    print("3.")
+    print("H0 " + Smu + " = " + Smu + "0")
+    print("H1 " + Smu + " != " + Smu + "0")
+    opcion = int(input("Opcion: "))
+    os.system("cls")
+    if opcion not in [1, 2, 3]:
+        print("Opcion incorrecta, vuelve a intentarlo.")
+        print("Presione cualquier tecla para continuar...")
+        os.system("pause")
+        os.system("cls")
+        menu_varianza_desconocida()
+        return
+    
+    print("Prueb de hipotesis de una varianza conocida")
+    print("H0 " + Smu + " = " + Smu + "0")
+    print("H1 " + Smu + " " + ["<", ">", "!="][opcion - 1] + " " + Smu + "0")
+    print("")
+    print("Introduce los datos:")
+    n = int(input("n: "))
+    xbarra = float(input(Sxbar + ": "))
+    xmu = float(input(Smu + ": "))
+    sigma = float(input(Ssigma + ": "))
```

Comment:
```suggestion
    s = float(input("s: "))
```
Para varianza desconocida es con s la formula, tenemos que leer al igual los n datos y sacar su desviacion estandar

Reply:
o bien, podemos ignorar leer los n datos y hacerlo de esta manera, como prefieras

Reply:
yo digo que la desviación si se la podemos pedir como input, el profe siempre nos las daba

Reply:
smnn